src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalFeatureInputArgumentCollection.java

```diff
@@ -16,7 +16,7 @@
  */
 public class OptionalFeatureInputArgumentCollection implements ArgumentCollectionDefinition {
 
-    @Argument(fullName = StandardArgumentDefinitions.FEATURE_LONG_NAME, shortName = StandardArgumentDefinitions.FEATURE_SHORT_NAME, doc = "One or more files containing features", optional = true)
+    @Argument(fullName = StandardArgumentDefinitions.FEATURE_LONG_NAME, shortName = StandardArgumentDefinitions.FEATURE_SHORT_NAME, doc = "file containing features", optional = true)
```

Comment:
is this change because the help system adds "one or more" or something like that?